lib/components/artist/related_artists/related_artist.js

```diff
@@ -12,13 +12,10 @@ import ImageView from '../../opaque_image_view';
 class RelatedArtist extends React.Component {
   render() {
     const artist = this.props.artist;
-    const {height, width} = Dimensions.get('window');
-    const imageWidth = (width - 60)/2;
-    const imageAspectRatio = 1.5;
     return (
-      <View style={{paddingBottom: 30, width: imageWidth}}>
-        <ImageView style={{ width: imageWidth, height: imageWidth/imageAspectRatio }}
-                   imageURL={artist.image.url} />
+      <View style={{paddingBottom: 30, width: this.props.relay.variables.width}}>
```

Comment:
Interesting, hadn’t considered that yet 👍 

I do wonder if this will clash when we take screen scale into consideration, not sure if `prepareVariables` actually changes the variables. But we’ll see that then.